### PR TITLE
Local Bugfix + Cloud getDevice Example

### PR DESCRIPTION
**Pull request by Ekristoffe**
The Local command wasn't working, some header information where missing.
Add an example for listing devices.

### CODE REVIEW

File: src/local.ts [hunk 1]
```diff
@@ -9,6 +9,8 @@ export class Local {
       baseURL: address,
       headers: {
         'Content-Type': 'application/x-www-form-urlencoded; charset=utf-8',
+        'accept': 'application/json',
```
[the remark this answers is not in the document]
Comment (Ekristoffe) [reply src/local.ts]: Fixed

File: src/local.ts [hunk 2]
```diff
@@ -9,6 +9,8 @@ export class Local {
...
+        'X-Requested-With': 'XMLHttpRequest',
       },
       timeout: 5000,
```
[the remark this answers is not in the document]
Comment (Ekristoffe) [reply src/local.ts:15]: Strange which url are you using ?
direct ip like 10.0.1.156 or name like Remo-mini000000.local ? 
mine respond in less than a second
Maybe a network latency problem (tested with a windows pc and a BSD server)

File: examples/list-devices.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+const NatureRemo = require('nature-remo')
+
+async function listDevices() {
+  // Instantiate Nature Remo client with an API token
+  const client = new NatureRemo.Cloud(process.env.NATURE_REMO_CLOUD_API_TOKEN)
+
+  // Get a list of the devices
+  const devices = await client.getDevices()
+  for (const device of devices) {
+    const { id, nickname, type } = device
```
[the remark this answers is not in the document]
Comment (Ekristoffe) [reply examples/list-devices.js]: Fixed, sorry for that

File: src/local.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (uetchy) [reply src/local.ts:15]: I was querying via DNS-SD (.local hostname resolution) and it was incredibly slow so having timeout 20secs would at least make sense for those using Bonjour or something like that for their home automation system.

**Review by uetchy — APPROVED** [review]

File: src/local.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (Ekristoffe) [reply src/local.ts]: Maybe for a future update we could make the timeout something which can be updated.

**Pull request merged**